cmake/modules/FindTRILINOS.cmake

```diff
@@ -257,6 +160,4 @@ DEAL_II_PACKAGE_HANDLE(TRILINOS
     OPTIONAL Trilinos_TPL_INCLUDE_DIRS
   CLEAR
     TRILINOS_CONFIG_DIR EPETRA_CONFIG_H SACADO_CMATH_HPP ${_libraries}
-    TRILINOS_SUPPORTS_CPP11 TRILINOS_HAS_C99_TR1_WORKAROUND SACADO_CONFIG_H
-    TRILINOS_CXX_SUPPORTS_SACADO_COMPLEX_RAD
```

Comment:
You have to keep these variables names in the call to `DEAL_II_PACKAGE_HANDLE`. The reason is that all those calls to `FIND_[...]` and `[...]_SOURCE_COMPILES` are indeed cached. (So if Trilinos gets disabled, the corresponding configuration has to be disabled as well).